pgatk/tests/test_clinvar/test_data_downloader.py

```
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import pytest
+
+from pgatk.clinvar.data_downloader import NcbiDataDownloader
+
+
+class TestNcbiDataDownloader:
+    """Tests for NcbiDataDownloader."""
+
+    def test_build_refseq_urls(self):
+        downloader = NcbiDataDownloader(output_dir="/tmp/test")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)